submissions/pozzitive11/memoty-pair-game/js/index.js

```diff
@@ -49,34 +49,38 @@ const gameContainer = document.querySelector(".game__container");
 gameContainer.innerHTML = gameCard.join("");
 const gameCards = document.querySelectorAll(".game__card");
 
-let counter = 0;
-let arr = [];
-
-function flipCard() {
-  if (this && this.classList.contains("game__card")) {
-    this.classList.add("flip");
-
-    arr.push(this);
-  }
-  checkArrLength();
-
-  winCOndition();
-}
+let numberOfMatches = 0;
+let ArrayOfFlippedCards = [];
```

Comment:
Violation of naming convention is JS. `camelCase` only. `PascalCase` only for constructors